### DISCUSSION TITLE
Please officially support Playwright integration

### DISCUSSION THREAD

**Discussion by barakbs1** [percy/percy-puppeteer#317] [Q&A] [answered]
Hi Percy team!

As a former Puppeteer user, and a current Playwright user, will you guys ever officially support Playwright integration? Currently I the following package: [(https://www.npmjs.com/package/percy-playwright)]

It does the job great, but it always better to have official support :)

**Comment by Robdel12** [comment]
Hey @barakbs1! I don't believe we'll add a playwright specific SDK since the API between Puppeteer and Playwright are almost exactly the same. It looks like the linked package renames everything from `puppeteer` to `playwright`

I'm not opposed to it 100% because it is a little obtuse to use this SDK for playwright, but creating that package seems like it's instant tech debt since it's 99.99% a copy-paste of this SDK.

**Comment by vdhpieter** [comment]
@Robdel12 it indeed works with just the puppeteer package but the typing doesn't match (so I have to use `as any`). Also maybe in the future things will diverge and with the growing adoption of playwright maybe already setting something up that for now just uses the same methods as puppeteer is a good idea? Then if specific fixes are needed they can be implemented much quicker :)

**Comment by Robdel12** [comment, marked as answer]
We ended up shipping essentially a duplicate of the puppeteer package: https://github.com/percy/percy-playwright
